lomap/dbmol.py

```diff
@@ -125,21 +126,27 @@ def __init__(self, directory, parallel=1, verbose='off',
             if not isinstance(display,  bool):
                 raise TypeError('The display flag is not a bool type')
 
+            if not isinstance(radial,  bool):
+                raise TypeError('The display flag is not a bool type')
             output_str=''
             display_str=''
+            radial_str=''
```

Comment:
Based on the code below I think later the radial_str will be appended to the names_str and an empty string would be good to indicate that the option is not turn on. So I think empty string is ok here.